src/BaseQuery.php

```diff
@@ -153,6 +156,30 @@ public function orById($id)
 		return $this;
 	}
 
+	private function addJoins(array $columns, ?string $joinType) {
```

Comment:
hm, proc je to private? asi bych to dal protected?

Reply:
Souhlas